lib/process-services/src/lib/content-widget/attach-file-widget.component.html

```diff
@@ -59,18 +59,17 @@
 </div>
 
 <div data-automation-id="adf-attach-widget-readonly-list">
-    <mat-list *ngIf="hasFile">
-        <mat-list-item class="adf-attach-files-row" *ngFor="let file of field.value">
+    <mat-list *ngIf="hasFile" role="list">
+        <mat-list-item class="adf-attach-files-row" *ngFor="let file of field.value" role="listitem">
             <img mat-list-icon class="adf-attach-widget__icon"
                  [id]="'file-'+file.id+'-icon'"
                  [src]="file.content ? getIcon(file.content.mimeType) : getIcon(file.mimeType)"
                  [alt]="mimeTypeIcon"
                  (click)="onAttachFileClicked(file)"
                  (keyup.enter)="onAttachFileClicked(file)"
-                 role="button"
```

Comment:
Are we sure that removing this role is correct? These elements have some click and keyboard functions attached

Reply:
Yes i saw click event are there for image & the file name too. But still click event is working.

Reply:
the role is not for clicks, it's for screen readers to find where the button is, whether it is there or not, the click events are not related to that

Reply:
Addressed comment. please check.

Reply:
why it's needed to touch the "role" at all? what does it fix?

Reply:
Machine is reading as 'undefined' button. We have image tag so i have used role img & aria label.